app/src/main/java/com/github/epfl/meili/poi/PoiActivity.kt

```diff
@@ -3,52 +3,54 @@ package com.github.epfl.meili.poi
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import androidx.fragment.app.Fragment
-import androidx.fragment.app.FragmentActivity
 import androidx.viewpager2.adapter.FragmentStateAdapter
 import androidx.viewpager2.widget.ViewPager2
 import com.github.epfl.meili.R
 import com.github.epfl.meili.map.MapActivity
+import com.google.android.gms.maps.model.PointOfInterest
```

Comment:
I'm still crashing when clicking from the map view because of this discontinuity of POI versions I think.

Reply:
It should be fixed now (I removed clicking on pois for now).